src/test/java/org/javarosa/xpath/test/XPathEvalTest.java

```diff
@@ -523,6 +523,10 @@ public void doTests() {
         testEval("id-compress(0, 'CD','','ABCDE',1)", null, ec, new XPathException());
         testEval("id-compress(0, 'CD','CD','ABCDE',1)", null, ec, new XPathException());
 
+        testEval("checksum('verhoeff','41310785898')", null, null, "4");
```

Comment:
Can you remind me how you determined the correct values of these outputs prior to running them in the code here?